src/editor/editorconfig.jsdoc

```diff
@@ -13,7 +13,7 @@
  * An object defining the editor configuration can be passed when initializing the editor:
  *
  *		EditorClass
- *			.create( {
+ *			.create( document.querySelector( '#editor' ), {
```

Comment:
This part of the documentation is not related to this ticket, and it needs bigger changes, so I think we should not touch it now.